src/sonic-config-engine/minigraph.py

```diff
@@ -285,6 +285,15 @@ def get_mgmt_info(devices, dev, port):
 
     return ret_val
 
+def get_alias_map_list(hwsku):
+    alias_map_json = os.path.join('/usr/share/sonic', hwsku, 'alias_map.json')
+    with open(alias_map_json) as data:
+        alias_map_dict = json.load(data)
+    alias_map_list = []
```

Comment:
when the alias map does not exists, let assume the sonic name is the same as origin name.